internal/runners/clean/uninstall.go

```diff
@@ -62,13 +63,30 @@ func (u *Uninstall) Run(params *UninstallParams) error {
 		return locale.NewError("err_uninstall_activated")
 	}
 
-	if !params.Force {
+	err := verifyInstallation()
+	if err != nil {
+		return errs.Wrap(err, "Could not verify installation")
+	}
+
+	if params.Prompt {
+		choices := []string{
+			locale.Tl("uninstall_prompt", "Uninstall the State Tool, but keep runtime cache and configuration files"),
+			locale.Tl("uninstall_prompt_all", "Completely uninstall the State Tool, including runtime cache and configuration files"),
+		}
+		selection, err := u.prompt.Select("", "", choices, new(string))
```

Comment:
The header for this command, "Uninstall" is already displayed, so there's no need to adorn the prompt with a header.